### PR TITLE
Add WuppieFuzz article

### PR DESCRIPTION
**Pull request by ThomasTNO**
Hi @tomootes,

This is a first draft for a [WuppieFuzz](https://github.com/TNO-S3/WuppieFuzz) article. Currently as blogpost. Not sure if this is the best spot, earlier today you mentioned the knowledge base would be suiting.

Any feedback and edits are welcome.

### CODE REVIEW

File: blog/draft/wuppiefuzz.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+authors: [thomas-rooijakkers]
+tags: [rest, api, security, security-by-design, validator, rust, tool, openapi, open-source, development, devops]
+draft: true
+---
+
+# WuppieFuzz: Coverage-Guided REST API Fuzzing voor Veiligere Digitale Overheidsdiensten
+In een tijd waarin digitale overheidsdiensten steeds vaker via [REST API’s](../2025/07/10/openapi-31-in-zicht#van-rest-naar-http) communiceren, is het waarborgen van de veiligheid van deze interfaces cruciaal. **[WuppieFuzz](https://github.com/TNO-S3/WuppieFuzz)**, ontwikkeld door [TNO](https://tno.nl), is een open-source, coverage-guided fuzzer die specifiek is ontworpen voor het testen van REST API’s. Het doel: kwetsbaarheden en bugs opsporen voordat ze misbruikt kunnen worden, met een sterke focus op gebruiksvriendelijkheid, modulariteit en inzichtelijkheid.
+
+![Logo of WuppieFuzz](https://raw.githubusercontent.com/TNO-S3/WuppieFuzz/459193a9e423b40b561d24bf2de9d9f85b14b887/assets/WuppieFuzz.svg)
```
Comment (grebnetiew) [inline blog/draft/wuppiefuzz.md:10]: Is het een idee om naar een tag te verwijzen? `https://raw.githubusercontent.com/TNO-S3/WuppieFuzz/refs/tags/v1.2.0/assets/WuppieFuzz.svg`

**Comment by tomootes** [comment]
@ThomasTNO thanks a lot for your first contribution! I added a knowledge base article to this branch as well. I think the combination of a blog and a knowledge base article is the way to go. 

That way we can promote your blog, and still consolidate the knowledge about WuppieFuzz in our knowledge base. Maybe you can add a call for contributers to the blog? Since thats what you're trying to achieve as well right?

File: blog/draft/wuppiefuzz.md (hunk 1 above)
Comment (tomootes) [reply blog/draft/wuppiefuzz.md:10]: @grebnetiew / @ThomasTNO i would recommend adding the .svg to "/blog/draft/img". Its small so that would be no problem.

File: blog/draft/wuppiefuzz.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<!-- truncate -->
+
+## Wat is WuppieFuzz?
+WuppieFuzz is gebouwd bovenop het [LibAFL-framework](https://github.com/AFLplusplus/LibAFL) en ondersteunt drie testmodi:
+- **Black box**: zonder kennis van de interne werking.
+- **Grey box**: met beperkte kennis en observatie van gedrag.
+- **White box**: met volledige toegang tot de broncode en interne structuur.
+
+
+Op basis van een [OpenAPI-specificatie](../../kennisbank/apis/openapi-specification) genereert WuppieFuzz automatisch zinvolle sequenties van HTTP-requests. Deze sequenties worden vervolgens gemuteerd om diepere logica en edge cases in de API te bereiken. De tool meet test coverage via responscodes en/of via instrumentatie van de backend (bijv. met `JaCoCo` voor Java of `coverage.py` voor Python).
```
Comment (sofotech) [inline blog/draft/wuppiefuzz.md:20]: responscodes -> response codes

**Comment by ThomasTNO** [comment]
> @ThomasTNO thanks a lot for your first contribution! I added a knowledge base article to this branch as well. I think the combination of a blog and a knowledge base article is the way to go.
> 
> That way we can promote your blog, and still consolidate the knowledge about WuppieFuzz in our knowledge base. Maybe you can add a call for contributers to the blog? Since thats what you're trying to achieve as well right?

Thanks for the suggestion. Added a call for contributors to the blog
